src/Bloc/BlElement.class.st

```diff
@@ -747,26 +753,34 @@ BlElement >> childIndexOf: anElement [
 { #category : #'children accessing' }
 BlElement >> childNamed: aChildName [
 	<return: #BlElement>
+	
+	self 
+		deprecated:  'Use childWithId: instead' 
+		transformWith: '`@receiver childNamed: `@statements1' -> '`@receiver childWithId: `@statements1'.
 
-	self
-		childNamed: aChildName
-		ifFound: [ :anElement | ^ anElement ]
-		ifNone: [ self error: 'Children container named ', aChildName asString, ' does not exist' ]
+	^ self childWithId: aChildName
 ]
 
 { #category : #'children accessing' }
 BlElement >> childNamed: aContainerName ifFound: ifFoundBlock [
-	<return: #Object>
+	<return: #BlElement>
+	
+	self 
+		deprecated:  'Use childWithId:ifFound: instead' 
+		transformWith: '`@receiver childNamed: `@statements1 ifFound: `@statements2' -> '`@receiver childWithId: `@statements1 ifFound: `@statements2'.
 
 	^ self
-		childNamed: aContainerName
+		childWithId: aContainerName
 		ifFound: ifFoundBlock
-		ifNone: [  ]
 ]
 
 { #category : #'children accessing' }
 BlElement >> childNamed: aContainerName ifFound: ifFoundBlock ifNone: ifNoneBlock [
-	<return: #Object>
+	<return: #BlElement>
```

Comment:
We are gradually removing this pragma. But it's ok to keep it in this PR.

Reply:
Ok ! I forgot to ask you about this pragma, so when I see this pragma I can remove ?

Reply:
Yes. However, I'm keeping them when it's not an obvious information (like if it a testing selector is*, then it's obvious it restuns a Boolean). For a BlElement, IMO, a user should assume `childWithId:` returns a BlElement.